include/bluetooth/services/bms.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BT_GATT_BMS_H_
+#define BT_GATT_BMS_H_
+
+/**@file
+ * @defgroup bt_gatt_bms Bond Management Service API
+ * @{
+ * @brief API for the Bond Management Service (BMS).
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdlib.h>
+#include <zephyr/types.h>
+
+/** @def BT_UUID_BMS
+ *  @brief Bond Management Service
+ */
+#define BT_UUID_BMS                       BT_UUID_DECLARE_16(0x181e)
+/** @def BT_UUID_BMS_CONTROL_POINT
+ *  @brief Bond Management Control Point
+ */
+#define BT_UUID_BMS_CONTROL_POINT         BT_UUID_DECLARE_16(0x2aa4)
+/** @def BT_UUID_BMS_FEATURE
+ *  @brief Bond Management Feature
+ */
+#define BT_UUID_BMS_FEATURE               BT_UUID_DECLARE_16(0x2aa5)
```

Comment:
This code section will be replaced by the following change:
https://github.com/zephyrproject-rtos/zephyr/pull/26468
Upstream PR is merged but I still need to wait to have it downstream (in NCS)